problem.py

```diff
@@ -25,50 +26,46 @@
 ]
 
 
-def get_cv(folder_X, y):
+def get_cv(X, y):
     # _, X = folder_X
     # cv = StratifiedShuffleSplit(n_splits=1, test_size=0.2, random_state=57)
     # return cv.split(X, y)
 
-    # for now dummy CV that has one fold with all data for both train/valid
-    return [(slice(None), slice(None)), ]
+    # quadrangles for training have shape:  4082, 3397, 2698 (total 10177)
+    # for each fold use one quadrangle as test set, the other two as training
+    return [(slice(0, 4082 + 3397), slice(4082 + 3397, None)),
+            (slice(4082, None), slice(None, 4082)),
+            (np.r_[0:4082, 4082+3397:10177], slice(4082, 4082 + 3397))]
```

Comment:
I now hardcoded the slices here for the training data. However, this is not really clean :-) 
And more importantly, will also not work when different data are used in the backend. 

Ideas on how to solve this?

We could add a small csv file in the `data` dir that holds this information? (`data_train_partitions.csv`) But, the `get_cv` function only gets the actual X and y data as input. 
Otherwise, we could add a column in the y data indicating it. But that also messes with the `y` format ..

Or we can decide we don't care about the exact division by quadrangles, and just split it in 3 equal parts ..

Reply:
I think numpy array can have metadata appended to them, it might be a possibility.. @kegl knows more about this.

I guess it also depends what we want to be robust against. Is it the flexibility of the model of predicting craters in different environments or just general accuracy, in which case we should shuffle and split in 3 parts (also easier).